UI/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset='UTF-8'>
+		<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+		<link rel="stylesheet" type="text/css" href="styles/styles.css">
+		<link href="https://fonts.googleapis.com/css?family=Anton" rel="stylesheet"> 
+		<link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope" rel="stylesheet"> 
+		<link rel="icon" href='images/icon.png'>
+		<title>Dorogram</title>
+	</head>
+	<body>
+		<div class='header'>
```

Comment:
1. Please, read about semantic html, the info should in lectures. You have quite structured and accurate html, but you are supposed to use semantic tags. Tag 'header' is necessary tag, you should also check tags <main>, <footer>, <article> and use it in your code
2. Usually only double quotes are used in html, it's better not to do this ``class='name-header'``